### PR TITLE
Green dev

### PR DESCRIPTION
**Pull request by paulfgraham**
### Description
The `green-dev` branch, which is hosted on https://green.dev.cop.homeoffice.gov.uk/.

This should only be merged once the QA and UCD teams are happy with it. Which they now are! 😁

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -30,63 +30,6 @@ steps:
     target:
     - secrets
 
-- name: synch_sit_secrets
```
Comment (paulfgraham) [inline .drone.yml:33]: Is removing this correct? Not familiar with what this change is for...

File: .github/workflows/build-docker.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: Publish Docker
 on:
   push:
     branches:
-      - master
+      - green-dev
```
Comment (paulfgraham) [inline .github/workflows/build-docker.yml:5]: Maybe this should be both `master` _and_ `green-dev`? 🤔

File: .github/workflows/ui.yml [hunk 3]
```diff
@@ -5,9 +5,9 @@ name: COP UI CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [ green-dev ]
   pull_request:
-    branches: [ master ]
+    branches: [ green-dev ]
```
Comment (paulfgraham) [inline .github/workflows/ui.yml:10]: Again, I think this maybe should be both branches...

**Comment by azee781** [comment]
**green-dev** _Kube_, _.github/workflows_ and _.drone_ config changes  must **not** be merged in **master**

please make sure only merge code changes from green-dev into master

**Comment by paulfgraham** [comment]
> **green-dev** _Kube_, _.github/workflows_ and _.drone_ config changes must **not** be merged in **master**
> 
> please make sure only merge code changes from green-dev into master

There's no easy way to do that. We'll have to rip your changes out, merge the PR, and then add your changes back in. Your changes are part of the codebase. 🤷‍♂️

File: .github/workflows/build-docker.yml (hunk 2 above)
Comment (paulfgraham) [inline .github/workflows/build-docker.yml:5]:
```suggestion
      - [ master, green-dev ]
```

**Review by hiteshprayosha — APPROVED** [review]

**Pull request merged**